### DISCUSSION TITLE
missing 4.1.0 release for micronaut-platform

### DISCUSSION THREAD

**Discussion by vensad** [micronaut-projects/micronaut-core#9704] [Q&A] [answered]
i was running into dependency resolution failures when trying to use 4.1.0 . 

wondering why there is no v4.1.0 for [micronaut-platform](https://github.com/micronaut-projects/micronaut-platform/releases) . i do observe v4.1.0 for [micronoaut-core](https://github.com/micronaut-projects/micronaut-core/releases). appreciate any help

**Comment by dstepanov** [comment]
There wasn’t a release yet

**Reply by sdelamo** [reply, marked as answer]
Micronaut Platform will update to Micronaut Core 4.1.0 or later in Micronaut Platform 4.1.0. We typically release minor versions 6 to 12 weeks after the previous minor. We release 4.0.0 July 14th. Thus you can expect Micronaut Framework 4.1.0 last week of August or first weeks of September

**Reply by vensad** [reply]
👍
